src/scraper.rs

```diff
@@ -177,6 +182,19 @@ impl Scraper {
         .unwrap();
     }
 
+    /// Sleep the thread for a variable amount of seconds to avoid getting banned
+    fn sleep(&self, rng: &mut rand::rngs::ThreadRng) {
+        let base = self.args.delay;
+        let delay_range = self.args.random_range;
+
+        if base == 0 && delay_range == 0 { return; }
+
+        // delay_range+1 because gen_range is exclusive on the upper limit
+        let rand_delay_secs = rng.gen_range(0, delay_range+1);
```

Comment:
```suggestion
        let rand_delay_secs = rng.gen_range(0, delay_range + 1);
```

Add more spaces between operators